src/extension/extension.ts

```diff
@@ -808,8 +808,14 @@ async function readExtension(
           [],
         );
         if (resolved.include.length > 0) {
-          (object.project as Record<string, unknown>)[key] = resolved
-            .include;
+          if (key === "brand") {
+            (object.project as Record<string, unknown>)[key] = relative(
+              join(extensionDir, "..", ".."),
```

Comment:
This is somewhat fishy - is the extension directory always the grandchild of the project directory?

Reply:
Indeed, this breaks when an extension is installed from GitHub, because the path will be of the form `_extensions/github-org/extension-name`. Checking...

Reply:
Resolution: search upward in path to find `_extensions`. Not pretty but safe for our purpose.